packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -108,76 +108,75 @@ function GridItem< Item >( {
 				selection={ selection }
 				onChangeSelection={ onChangeSelection }
 				getItemId={ getItemId }
-				primaryField={ primaryField }
+				primaryField={ titleField }
```

Comment:
Should we just remove the primaryField concept and call it titleField everywhere?

Reply:
probably yes, I'll make the update